client/src/components/utils/Button/index.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './style.css';
+
+export default function Button({ className, children }) {
```

Comment:
You need to receive the `onClick` listener, for adding functionality to the button